test/JDBC/README.md

```diff
@@ -1,5 +1,377 @@
+### SQL Authentication
+To check different authentication use cases via JDBC SQL Server Driver:
+```
+java_auth#!# < connection attribute and value pairs follow, separated by '#!#' delimiter>
+```
+
+The connection attribute and value are to be mentioned as pairs:
+```
+<connection attribute> |-| <value>
+```
+
+List of valid connection property types and values are [here](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/connect/jdbc/setting-the-connection-properties.md#properties)
+
+**Example**
+```
+java_auth#!#databaseName|-|master
+java_auth#!#user|-|dummy#!#password|-|hello#!#databaseName|-|demo
+```
+
+Input file type: `.txt`
+
+---
+
+### Cross Dialect (intermixing queries in T-SQL and PL/pgSQL dialect)
+A SQL Batch in T-SQL should be added as:
+```tsql
+-- tsql <T-SQL connection attribute and value pairs if any, separated by spaces>
+/* T-SQL batch */
+GO
+```
+
+A SQL Batch in PL/pgSQL should be added as:
+```tsql
+-- psql <PSQL connection attribute and value pairs if any, separated by spaces>
+/* PL/pgSQL batch */
+GO
+```
+
+**NOTE:** Even though `GO` is not a batch separater in PL/pgSQL, it needs to added because the test framework uses it as a delimiter internally to process these SQL batches.
+
+Connection attributes should be specified in pairs:
+```
+<connection attribute 1>=<value 1>     <connection attribute 2>=<value 2> ...
+```
+
+Currently the attributes that can specified are `user`, `password`, `database` and `currentSchema` (only for PG connection).
+
+**Example**
+```tsql
+-- tsql
+CREATE PROCEDURE tsql_interop_proc1
+AS
+UPDATE procTab1 SET c1=10 where c2='b';
+INSERT INTO procTab1 values(1,'a');
+INSERT INTO procTab1 values(2,'b');
+EXEC psql_interop_proc2;
+GO
+
+-- psql     currentSchema=master_dbo,public
+CREATE PROCEDURE psql_interop_proc1()
+AS
+$$
+BEGIN
+    CREATE TABLE procTab1(c1 int);
+    INSERT INTO procTab1 values (5);
+    ALTER TABLE procTab1 ADD c2 char;
+    CALL tsql_interop_proc1();
+END
+$$ LANGUAGE PLPGSQL;
+GO
+
+CREATE PROCEDURE psql_interop_proc2()
+AS
+$$
+BEGIN
+    INSERT INTO procTab1(c1,c2) values (3,'c');
+    UPDATE procTab1 SET c1=10 where c2='b';
+    INSERT INTO procTab1(c1,c2) values (4,'d');
+    DELETE FROM procTab1 where c2='a';
+END
+$$ LANGUAGE PLPGSQL;
+GO
+
+-- tsql
+EXEC psql_interop_proc1
+GO
+SELECT * FROM procTab1 order by c1
+GO
+
+-- psql     currentSchema=master_dbo,public
+CALL tsql_interop_proc1()
+GO
+SELECT DISTINCT c1 FROM procTab1
+GO
+```
+
+Input file type: `.mix`
+
+---
+
+### **IMPORTANT**
+- If you want to execute a SQL Batch in `.txt` input files, you will need to specify the batch in a single line without the `GO` batch separator. This is needed because for `.txt` files, the test framework treates every line as a standalone statement/command that can be executed against the server.
+- You CANNOT club functionalities of different file types. For example, you cannot execute prep-exec statements (functionality of `.txt` input file) in a `.mix` file. 
```

Comment:
I'm not totally I understand what you're saying, but does this capture the meaning:

You CANNOT group functionalities from a different file type. For example, you cannot execute prep-exec statements (the functionality of a `.txt` input file) in a `.mix` file.